projects/Terminal_Based_Hangman_Game/hangman.py

```diff
@@ -0,0 +1,180 @@
+import random
+import json
+
+#function to randomly get one word from words.py and convert the word to uppercase
+def get_word():
+    with open('words.json') as json_file:
+        data = json.load(json_file)
+    
+    wordArray = data["word_list"]
+    word = random.choice(wordArray)
+    return word.upper()
```

Comment:
do not manipulate string in return
```suggestion
    word = word.upper()
    return word
```